config/src/main/java/com/alibaba/nacos/config/server/utils/SimpleIPFlowData.java

```diff
@@ -23,11 +23,11 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 /**
- * 根据IP进行流控, 控制单个IP的数量以及IP总量
+ * According to IP flow control, control the number of individual IP and IP total.
  *
  * @author leiwen.zh
  */
-@SuppressWarnings("PMD.ClassNamingShouldBeCamelRule")
+@SuppressWarnings({"PMD.ClassNamingShouldBeCamelRule", "checkstyle:AbbreviationAsWordInName"})
```

Comment:
Why add this SuppressWarnings？

Reply:
There are more than two consecutive uppercase letters in the class name, but IP is generally indicated with uppercase letters.

Reply:
Ip is ok, I think.